### PR TITLE
Allow configuring redirect handler using the @ClientRedirectHandler annotation

### PR DESCRIPTION
**Pull request by Sgitario**
Follow-up task of https://github.com/quarkusio/quarkus/pull/31142#issuecomment-1429447206.

### CODE REVIEW

**Comment by Sgitario** [comment]
@geoand I guess these changes are what you had in mind in your comment https://github.com/quarkusio/quarkus/pull/31142#issuecomment-1429447206, right?

File: extensions/resteasy-reactive/rest-client-reactive/deployment/src/main/java/io/quarkus/rest/client/reactive/deployment/RestClientReactiveProcessor.java [hunk 1]
```diff
@@ -557,6 +492,108 @@ void addRestClientBeans(Capabilities capabilities,
         }
     }
 
+    private void populateClientExceptionMappersFromAnnotations(BuildProducer<GeneratedClassBuildItem> generatedClasses,
```
Comment (geoand) [inline extensions/resteasy-reactive/rest-client-reactive/deployment/src/main/java/io/quarkus/rest/client/reactive/deployment/RestClientReactiveProcessor.java:495]: The method name probably needs to be updated :)

**Review by geoand — COMMENTED** [review]
Very nice!

I just added one small comment

File: extensions/resteasy-reactive/rest-client-reactive/deployment/src/main/java/io/quarkus/rest/client/reactive/deployment/RestClientReactiveProcessor.java (hunk 1 above)
Comment (Sgitario) [reply extensions/resteasy-reactive/rest-client-reactive/deployment/src/main/java/io/quarkus/rest/client/reactive/deployment/RestClientReactiveProcessor.java:495]: I've renamed it to `populateClientExceptionMapperFromAnnotations`. 
We have two methods:
- `populateClientExceptionMapperFromAnnotations` to populate `@ClientExceptionMapper`
- `populateClientRedirectHandlerFromAnnotations` to populate `@ClientRedirectHandler`

**Review by geoand — APPROVED** [review]
💪🏼

**Pull request merged**